### ISSUE TITLE
Investigate building books with CI/CD

### ISSUE THREAD

**Issue by choldgraf** [ExecutableBookProject/cli#6]
Something many have asked for in Jupyter Book is to use various CI/CD deployments for building and hosting books. We should look into this for whatever our "final" book build system is.

A few that we should explore:

* TravisCI
* CircleCI
* GitHub Actions
  * https://github.com/ammaraskar/sphinx-action
* Netlify
* Readthedocs

**Comment by jstac** [comment]
Yes, for sure, and perhaps even multiple options in order to be most helpful for [a range of users](https://github.com/ExecutableBookProject/meta/wiki/User-personas).
